### PR TITLE
add InfluxDB Enterprise 1.8 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
add InfluxDB Enterprise 1.8 release notes

### CODE REVIEW

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -8,18 +8,30 @@ menu:
...
-### Bugfixes
-- Update `MaxShardGroupID` on meta update.
-- Don't reassign shard ownership when removing a data node.
+#### **Back up meta data only**
+
+- Add option to back up **meta data only** (users, roles, databases, continuous queries, and retention policies) using the new `-strategy` flag and `only meta` option: `influx ctl backup -strategy only meta </your-backup-directory>`.
+
+    > **Note:** To restore a meta data backup, use the `restore -full` command and specify your backup manifest: `influxd-ctl restore -full </backup-directory/backup.manifest>`.
+
+#### **Incremental and full backups**
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:25]: Reading [this issue](https://github.com/influxdata/plutonium/pull/3061), not clear whether `influx ctl backup -incremental/-full` still works

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -8,18 +8,30 @@ menu:
...
+
+- Add `incremental` and `full` backup options to the new `-strategy` flag in `influx ctl backup`:
+  - `influx ctl backup -strategy incremental`
+  - `influx ctl backup -strategy full`
+
+### Bug fixes
+
+- Update the Anti-Entropy (AE) service to ignore expired shards.
+- Prevent a nonce mismatch (for example, when an `influx` process is restarted many times in short succession) by reducing the nonce specificity when checking an Azure Marketplace entitlement.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:34]: Does "checking an Azure Marketplace entitlement" mean same as "validating InfluxDB Enterprise on Azure Marketplace"?

File: content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (gianarb) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:25]: Hello! Yes the old one still work. The change is additive

Comment (kelseiv) [reply content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md:25]: thank you @gianarb!

**Pull request merged**